Client.Linq/Internal/QueryAggregator.cs

```diff
@@ -334,6 +347,16 @@ private string BuildFilter(IEnumerable<string> filterBy)
             return filter.ToString();
         }
 
+        private string BuildImports()
+        {
+            if (_imports != null)
+            {
+                return string.Join("", _imports.Select(x => "import \"" + x + "\"\n"));
+            }
+
+            return string.Empty;
```

Comment:
Please change implementation to:

```suggestion
            if (_imports.Count == 0)
            {
                return string.Empty;
            }

            return string.Join("", _imports.Select(x => "import \"" + x + "\"\n"));
```

it will be same in other parts of `QueryAggregator`.